### ISSUE TITLE
Role attribute of search results (i.e. 9 results found for [term]) should have role=alert prescribed to it so screenreaders know.

### ISSUE THREAD

**Issue by sidramatik** [CityofToronto/wp-open-data-toronto#26]

**Comment by sidramatik** [comment]
Instead of `<div id="results-count" aria-label="Number of dataset search results" style=""><span>4 datasets found for "sports"</span></div>`, should be:

`<div id="results-count" aria-label="Data catalogue search results">
<span role="alert">4 datasets found for "sports"</span>
</div>`

**Comment by sidramatik** [comment]
Clearer explanation than WCAG documentation:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role
